.gitignore

```diff
@@ -31,3 +31,5 @@ docker-compose.override.yml
 backend-tests/acceptance.*
 backend-tests/report_backend_integration_enterprise.html
 backend-tests/results_backend_integration_enterprise.xml
+
+.gitattributes
```

Comment:
Why? This is a very common file to have in a repository. There is a separate place for "your own" gitattributes, they should be put in `$GIT_DIR/info/attributes`.